lib/worker.sh

```diff
@@ -403,6 +412,11 @@ function TestCodebase() {
     #######################
     ERROR_CODE=$?
 
+    ########################################
+    # Increment counter that check was ran #
+    ########################################
+    (("TESTS_RAN++"))
+
```

Comment:
Refactor a bit to be done once rather than 4x times

Reply:
do you need the quotes on this?

Reply:
I used as similar approach for the `INDEX` variable:
- https://github.com/github/super-linter/blob/master/lib/worker.sh#L665

Otherwise, there will be an shell-lint defect in:
- https://github.com/v1v/super-linter/blob/dd06a64592e241612cb0ef43c779091acaf69077/lib/worker.sh#L490

Since it required a double quote in the conditional.

Therefore, I used the approach from the implementation to normalise the implementation in the test-executions.